### DISCUSSION TITLE
One Tap method to get back to the homepage in any case

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2826] [Mobile]
In the app, please do an experiment: just keep on following links deeper and deeper.
Okay. Now try to get back to the homepage.

Sure There is always a left pointing arrow in the upper left corner of the screen. Tapping it enough will eventually return one to the homepage. However that might require 15 taps. Therefore there should always be a button somewhere on the screen get one back to the homepage. (I have noticed the same problem with the official Twitter app also.)
The rest of the apps that I have used always have a way to get back to the homepage with one tap.

There is always room, just to the right of the arrow that points left in the left corner at the top of the screen. So how about putting a little home button icon there also?

Yes, as a workaround one can always just simply close the app and relaunch it. That way one is right back to the home page right away.

Note That when I'm talking about tapping more and more links I'm talking about internal GitHub links and nothing external. For instance going to an issue and then going to another issue that's referred from that issue and then going to a pull request that is referred from that and excetera excetera.

**Comment by candyho** [comment]
Thanks for sharing your feedback. I'll reflect this feedback to the team and see what we can do here.

**Reply by heaths** [reply]
This continues to be frustrating. Maybe just show the buttons all the time? Or have people swipe up from the bottom of the app much like Android does in full screen apps to show it's system buttons.

**Reply by heaths** [reply]
Over holidays, I've been exploring many different repos that opening from mastodon or wherever just push more projects/source into the history stack, making it harder and harder to get back home to check notifications or whatever.

At this point, either uninstalling the app or unsetting it as the default handler for URLs will be more productive.

At the very least, a hamburger menu would take minimal screen real estate and give you options for plenty of global navigation targets like home, navigation, my PRs and issues, etc.

**Reply by doronz88** [reply]
I'm also encountering the same issue. Though viewing stuff from the mobile app is much smoother, the lack of a simple home button makes the whole experience a nightmare. 

If I want to search for a repo for example, I have to go to my browser, search for the repo and only there click on the link the triggers the app.

Not having the ability to search for projects or access the homepage is a HUGE letdown making the whole experience really frustrating and the app irrelevant (except as an authentication method)

**Reply by matglas** [reply]
💯% agree with the experience of others and sorry to see this feedback be open for 3y.

I recently wanted to research a repo on a protocol that linked into other repositories a lot. It was a pain.

Having the option to home is 1. But being able to quickly switch between repos that you had open would be even better. Much more like a browser experience.

But please start with a home button.

**Reply by meichthys** [reply]
@candyho is there any update on this?
